src/styles/components/nodes-table/styles.less

```diff
@@ -0,0 +1,13 @@
+& when (@nodes-table-enabled) {
+
+  .nodes-table {
+
+    col.node-table--col-health {
```

Comment:
Same observation about the unnecessary specificity `col.node-table--col-health` which could be written as `node-table--col-health`.

Not big deal though and I wouldn't block the PR

Reply:
missed this while cleaning up - but you're right.